spacelift/internal/structs/bitbucket_cloud_integration.go

```diff
@@ -0,0 +1,6 @@
+package structs
+
+// BitbucketCloudIntegration represents bitbucket cloud integration.
+type BitbucketCloudIntegration struct {
```

Comment:
It looks like these structs are just used in the graphql queries and no-where else. Given that, it might be simpler to just define them inline in the queries - that way you can see the full query in one place.